ami/main/migrations/0079_userprojectmembership_migration.py

```
@@ -0,0 +1,106 @@
+from django.db import migrations, models
+from django.conf import settings
+import django.db.models.deletion
+
+
+def forwards(apps, schema_editor):
+    Project = apps.get_model("main", "Project")
```

Ruff (0.14.7)

Local variable `Project` is assigned to but never used

Remove assignment to unused variable `Project`

(F841)